content/episode/zee-spencer-cohere-zinc-coop.md

```diff
@@ -0,0 +1,619 @@
+
+{{< timestamp "[00:16:26]" >}}
+But it's really hard to sell that compared to "I can keyboard-hands Rails applications." And it just wasn't particularly fulfilling to like, feel like if we wanted to be successful, we had to give up on what we were trying to be and what we're trying to do.
+
+{{< timestamp "[00:16:41]" >}}
+So, in, I think it must've been early 2019, yeah, early 2019 Betsy started working at the Long-Term Stock Exchange. Because we're structured in this belief system where like, hey, we want to care for each other and build this long-term asset and this value, we didn't do what you would traditionally do in a startup and be like, "Hey, you're gone. We're going to claw back your equity or you need to give us a chunk of cash." We're just like, no, like, you can hold onto what you've got and it's totally fine. It's important for us to preserve the asset that we have built together. And so she's still working at LTSE.
+
+{{< timestamp "[00:17:12]" >}}
+And, Jennifer and I switched more into the okay, cool, now that we're in a spot where we're sustainable and we're comfortable, cause we didn't have three people all working together, on a budget that couldn't support us, what do we want to do? and we, you know, we had a nice relaxing year.
+
+{{< timestamp "[00:17:29]" >}}
+We made our clients pretty happy. But at the same time, I was like, Cohere isn't the right vehicle for building longterm value for it, from the idea of like a product or a service, right? If you're a consultancy, that's mostly focusing on selling your time and attention, you don't have the same ability to focus on product development or, or customer development in that way.
+
+{{< timestamp "[00:17:51]" >}}
+And you kind of dilute the brand, you dilute the attention. And I'm like, "Well, what if I really kind of like doubled down on learning this cooperative stuff?" And so '18 was when I first started learning about the Sustainable Economies Law Center, the East Oakland Collective, Repaired Nations, all of these organizations in Oakland, California that were really focusing on building cooperative economics locally.
+
+{{< timestamp "[00:18:12]" >}}
+And I was like "THAT" like,  the East Bay Permanent Real Estate Cooperative was exactly what I was wanting to exist, but for technology companies. And so through the course of 2018, I was just looking at it like, okay. So what they do is they buy assets and they maintain those assets. And then they sell those assets to the people who live there.
+
+{{< timestamp "[00:18:30]" >}}
+Well, what if we did that with software products that no longer have a good home? So I bought an app from a friend of a friend and I'm like, Hey, this brings in about a thousand dollars a quarter, not a lot of money, but if we just buy it off you and then do a revenue share for a few years and we maintain it, we release updates. How does that sound? I'm like, yeah, that sounds great. Cause I'm not maintaining it. I'm not releasing updates.
```

Comment:
This probably should be "He's like, yeah, that sounds great." I mis-spoke :).

Reply:
Actually I think the transcript is just wrong, you are not saying "I'm" in that bit. Fixed.